Formula/h/hiredis.rb

```diff
@@ -1,11 +1,21 @@
 class Hiredis < Formula
   desc "Minimalistic client for Redis"
   homepage "https://github.com/redis/hiredis"
-  url "https://github.com/redis/hiredis/archive/refs/tags/v1.2.0.tar.gz"
-  sha256 "82ad632d31ee05da13b537c124f819eb88e18851d9cb0c30ae0552084811588c"
   license "BSD-3-Clause"
   head "https://github.com/redis/hiredis.git", branch: "master"
 
+  stable do
+    url "https://github.com/redis/hiredis/archive/refs/tags/v1.2.0.tar.gz"
+    sha256 "82ad632d31ee05da13b537c124f819eb88e18851d9cb0c30ae0552084811588c"
+
+    # Makefile: correctly handle version suffixes on macOS
+    # Remove with `stable` block on next release.
```

Comment:
Instructions unclear, stuck in `stable`🐴  🤠

Reply:
Aha, good point, but it seems hiredis is staying on the BSD license? I guess we can re-evaluate this when the next release happens.

https://github.com/redis/hiredis?tab=BSD-3-Clause-1-ov-file